packages/react-components/react-combobox/src/components/Option/useOptionStyles.ts

```diff
@@ -17,11 +17,12 @@ const useStyles = makeStyles({
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
     color: tokens.colorNeutralForeground1,
     columnGap: tokens.spacingHorizontalXS,
-    cursor: 'default',
+    cursor: 'pointer',
     display: 'flex',
+    fontFamily: tokens.fontFamilyBase,
     fontSize: tokens.fontSizeBase300,
     lineHeight: tokens.lineHeightBase300,
-    ...shorthands.padding(tokens.spacingHorizontalSNudge),
+    ...shorthands.padding(tokens.spacingHorizontalS),
```

Comment:
Should this be horizontal-only padding?
```suggestion
    ...shorthands.padding(0, tokens.spacingHorizontalS),
```

Or if it was meant to be both, then use the vertical tokens for vertical padding:
```suggestion
    ...shorthands.padding(tokens.spacingVerticalS, tokens.spacingHorizontalS),
```